### ISSUE TITLE
Move the client access check to the server-side

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdGuardHome#1925]
@szolin the only viable solution I see is to supply this information from the server-side. This is the only way to make it work fast enough.

Here's what I propose:

1. Change the `/control/clients/find` method, add information about whether this client is blocked or not (and if blocked - by what rule). If the "Allowed clients" list is not empty, and the client is not listed there, this needs to be indicated as well.
2. On the UI side, use this information to indicate client access status.
3. "Unblock" method should remove the corresponding item from the disallowed clients list (even if it's a CIDR).
4. If the client is not allowed because it's missing from the "Allowed clients" list, "Unblock" button must be shown, but be in the "disabled" state. Tooltip text: "The client is not allowed because it is not in the "Allowed clients" list."

_Originally posted by @ameshkov in https://github.com/AdguardTeam/AdGuardHome/issues/1920#issuecomment-663435807_

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdGuardHome/issues/2075
